src/charactersheet/viewmodels/common/player_image/index.html

```diff
@@ -21,9 +21,6 @@
     <!-- /ko -->
   </div>
 </div>
-<div class="connection-status">
-  <div class="circle-indicator" data-bind="css: statusIndicatorClass"></div>
```

Comment:
Is this getting moved? or removed?

Reply:
They were removed. I wanted to add them back after our convo. Haven't done it yet.